README.md

```diff
@@ -91,7 +91,7 @@ on your project.
 
 ## Limitations
 
-This tool can be used in the following scenarii :
+This tool can be used in the following scenaro :
```

Comment:
```suggestion
This tool can be used in the following scenario :
```

Reply:
Regarding [wikiperia](https://en.wiktionary.org/wiki/scenarii) : 

> However, the modern plural of the Italian word [scenario](https://en.wiktionary.org/wiki/scenario#Italian) is [scenari](https://en.wiktionary.org/wiki/scenari#Italian).

So the fix should be `scenari` IMHO (I originally meant to write the plural of scenario, hence the two i's, but it looks like I wrote it wrong).

Reply:
Oh I got it wrong, from the PR I didn't get there were multiple ones!

Reply:
However, the modern plural of the Italian word scenario is scenari. (not in english)